app/views/episodes/media/_complete.html.erb

```diff
@@ -9,7 +9,7 @@
     </button>
 
     <div class="form-control d-flex align-items-center">
-      <div class="text-secondary me-2"><%= media.file_name %></div>
+      <div class="text-secondary me-2 flex-grow-1 overflow-hidden text-truncate"><%= media.file_name %></div>
```

Comment:
In Media fields, text was overflow the space if the name was really long. Since these are not Inputs, we need to truncate the text.